File: src/Example3Java/EX06.java
package Example3Java;

import java.util.Scanner;

public class EX06 {
    public static void main(String[] args) {
        System.out.print("Enter Text : ");
        String str = new Scanner(System.in).nextLine();
        int num = 0;
        int index = str.indexOf("f");
        for (int i = 0; i < str.length(); i++)
            if (str.charAt(i) == 'F' || str.charAt(i) == 'f')
                num += 1;
        System.out.println("Text : "+str+" , sum 'F' and 'f' = "+num);
    }
}

File: src/Example3Kotlin/EX06.kt
import java.util.*

object EX06 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter Text  : ")
        val str = Scanner(System.`in`).nextLine()
        var num = 0
        for (i in 0..str.length -1) {
            if (str[i] == 'F' || str[i] == 'f')
                num += 1
        }
        println("Text : $str , sum 'F' ad 'f' = $num")
    }
}
